src/components/EditContactInformation/EditContactInformation.jsx

```diff
@@ -0,0 +1,181 @@
+// type,
+//       name,
+//       street,
+//       zipCode,
+//       state,
+//       qbVendorName,
+//       qbCityStateZip,
+//       primaryPhone,
+//       backupPhone,
+//       primaryEmail,
+//       comments,
+//       faxPhone,
+//       contactName,
+//       website,
+//       businessHours,
+//       findOut,
+//       onboardingStatus,
+//       joinDate,
+//       inputTypeStatus,
+//       vendorType,
+//       status,
+//       petsOfTheHomelessDiscount,
+//       updatedBy,
+//       updatedDateTime,
+//       syncToQb,
+//       veterinary,
+//       resource,
+//       food,
+//       donation,
+//       familyShelter,
+//       wellness,
+//       spayNeuter,
+//       financial,
+//       reHome,
+//       erBoarding,
+//       senior,
+//       cancer,
+//       dog,
+//       cat,
+//       fphPhone,
+//       contactPhone,
+//       webNotes,
+//       internalNotes,
+//       published,
+//       shelter,
+//       domesticViolence,
+//       webDateInit,
+//       entQb,
+//       serviceRequest,
+//       inactive,
+//       finalCheck,
+//       createdBy,
+//       createdDate,
+//       city,
+
+import { useBackend } from '../../contexts/BackendContext';
+import { useState, useEffect } from 'react';
+import { Flex, Text, Textarea, Stack, Box, Spacer, Button, Input, InputGroup, InputLeftElement, Icon } from '@chakra-ui/react';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Spacer' is defined but never used.